app/jobs/review_users.py

```diff
@@ -34,7 +35,7 @@ def add_notification_reminders(db: Session, user: "User", now: datetime = dateti
                 user.id, user.expire)
 
 
-def review():
+def review(bg: BackgroundTasks):
```

Comment:
Passing a bg instance isn't a good idea. as mention [here](https://www.starlette.io/background/), if one of the tasks raises an exception, the next tasks will not execute. so I would rather use threading for this, or at least have an instance of BackgroundTasks in each run. so please change this section.